.github/workflows/publish-collection.yml

```diff
@@ -1,6 +1,6 @@
 ---
 name: Publish Ansible Collection
-'on':
+"on":
```

Comment:
Release train will revert this

Reply:
Cheers, I'll remove changes to this file.